plugins/database/postgresql/postgresql.go

```diff
@@ -77,7 +79,8 @@ func new() *PostgreSQL {
 type PostgreSQL struct {
 	*connutil.SQLConnectionProducer
 
-	usernameProducer template.StringTemplate
+	usernameProducer   template.StringTemplate
+	PasswordEncryption passwordAuthentication
```

Comment:
Not sure if this is used outside of this directory, but would be good to update this to `PasswordAuthenticator` as well.